docs/DEPLOY_WEBINY_PROJECT_CF_TEMPLATE.yaml

```diff
@@ -40,6 +40,11 @@ Resources:
                   "Null":
                     "aws:ResourceTag/WbyProjectName": "false"
 
+              - Effect: Allow
+                Resource: arn:*:cloudfront::*:distribution/*
+                Action:
+                  - cloudfront:GetDistribution
```

Comment:
`GetDistribution` is included in the policy before this one, but I think the Condition is failing. This "*" resource filter is perhaps too permissive.